database/BUILD

```diff
@@ -45,6 +45,7 @@ native_java_libraries(
     deps = [
         # External dependencies from Vaticle
         "@vaticle_typedb_common//:common",
+        "@vaticle_typeql//java:typeql-lang",
```

Comment:
Required in order for the schema export to be done in the `CoreDatabase` directly. I believe this is a sensible dependency to have and there is no good alternative location for this dependency to live.

Explored the alternative of having the parsing in `ConceptManager` and `LogicManager` separately, but this leads to having to combine the parsed definitions in a way that could affect correctness of the final schema, so we leave eliminate this option.